public/js_v2.0.1_f1/ai_chat_functions.js

```diff
@@ -206,6 +222,24 @@ async function buildRequestObjectForAiConv(msgAttributes, messageElement = null,
             messageObj.iv = cryptoMsg.iv;
             messageObj.tag = cryptoMsg.tag;
 
+            // handle auxiliaries data
+            messageObj.auxiliaries = messageObj.auxiliaries ?? []
+            auxiliaries = []
+            for (const auxiliary of messageObj.auxiliaries) {
+                const cryptoMsg = await encryptWithSymKey(convKey, auxiliary.content, false);
```

Comment:
Each auxiliary is encrypted on the client side before being passed to the server for storage, just like the the message content.